nanoFramework.Device.Bluetooth/BluetoothLEServer.cs

```diff
@@ -0,0 +1,267 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Collections;
+using nanoFramework.Device.Bluetooth;
+using nanoFramework.Device.Bluetooth.GenericAttributeProfile;
+using System.Diagnostics;
+
+namespace nanoFramework.Device.Bluetooth
+{
+    /// <summary>
+    /// Represents a Bluetooth LE Server.
+    /// </summary>
+    public class BluetoothLEServer : IDisposable
+    {
+        private bool disposedValue;
+        private DevicePairing _pairing;
+        private GattSession _session;
+
+        private static BluetoothLEServer _instance;
+        private static readonly object _lock = new object();
+
+        internal static readonly BluetoothEventListener _bluetoothEventManager = new();
+        internal static ArrayList _services = new();
+
+        static BluetoothLEServer()
```

Comment:
```suggestion
        private static BluetoothLEServer()
```

Reply:
Can't have access modifier on static constructor., which makes sense.  It gives an error.

Reply:
ho, okok!